libevmasm/Assembly.cpp

```diff
@@ -756,3 +759,31 @@ LinkerObject const& Assembly::assemble() const
 	}
 	return ret;
 }
+
+vector<size_t> Assembly::decodeSubIds(u256 _data)
+{
+	size_t length = static_cast<size_t>((_data >> 248) & 0xFF);
+	if (length == 0)
+		length = 1;
```

Comment:
There is still some cases where old way of encoding is used. This is left to support these cases. Otherwise I need to find and change all of these places :)

Reply:
Oh, I though it was just a `size_t` before and the encoding was only introduced in this PR?